v5.5.sed

```diff
@@ -0,0 +1,6 @@
+s/@@VERSION@@/5.5/
+s|@@IMAGE@@|openshift/mysql-55-centos7|
```

Comment:
you could consistently use the same separator symbol, e.g., ",": `s,@@IMAGE@@,...,`